docs/source/configuration.rst

```diff
@@ -155,6 +155,9 @@ Example owner & admin configurations::
     admin_accounts =
             Exirel
             HumorBaby
+    # ignored when owner_account is set,
+    # but MUST NOT be empty
+    owner = DoesNotMatter
```

Comment:
```suggestion
    owner = dgw
```

I'd not let people put a "wrong" owner name, they might trip on that latter. You know how people work: they don't. They are broken and they make the most infuriating errors all the times. Let's not help them doing more harm.

Even tho I totally get the "DoesNotMatter". 😁